shared/fa2_modules/test/simple_admin_wrapper.mligo

```diff
@@ -0,0 +1,34 @@
+(*
+  Wrapper around simple_admin for testing:
+  embeds simple_admin.mligo and includes `unit` entrypoints
+  that trigger `fail_if_not_admin`, `fail_if_paused`
+*)
+
+#include "../simple_admin.mligo"
+
+type wrapper_storage = {
```

Comment:
Since the wrapper storage does not have any other data, consider just using plain admin storage: `type wrapper_storage = simple_admin_storage`. But this is minor point and you can leave it as is.